returnn/tf/layers/base.py

```diff
@@ -452,6 +452,11 @@ def transform_config_dict(cls, d, network, get_layer):
     which should be resolved.
     """
     from .basic import get_loss_class
+    if "from" not in d:
+      if BehaviorVersion.get() >= 1:
+        assert False, "Missing \"from\" in layer definition: %s/%s" % (network.name, d.get("_name", "<UNKNOWN>"))
+      else:
+        log.print_deprecation_warning("Not specifying \"from\" in layer definitions is deprecated.", behavior_version=1)
```

Comment:
The code looks a bit redundant.
I wonder whether we should introduce some generic helper, because code snippets like this will become common for other usages of `BehaviorVersion`.
Sth like this:

```suggestion
    BehaviorVersion.require(
      check="from" in d,
      message='%s/%s: missing "from"' % (network.name, d.get("_name", "<UNKNOWN>")),
      version=1)
```

Or so.

That `BehaviorVersion.require` function would basically do the logic you had before. Like:
```
@classmethod
def require(cls, check, message, version):
  if cls.get() >= version:
    raise Exception(...)
  else:
    log.print_exception_warning(...)
```